core/pkg/model/kubemodel/cluster.go

```diff
@@ -0,0 +1,28 @@
+package kubemodel
+
+import (
+	"time"
+)
+
+// TODO: do we need (start, end) for Cluster?
```

Comment:
@nikovacevic to address this question -- might be nice to have a metric for `cluster_info` that gives us UID and (start, end) if it's not too much work?